cves/CVE-2013-2135.yml

```diff
@@ -172,8 +194,8 @@ lessons:
     applies:
     note:
   distrust_input:
-    applies:
-    note:
+    applies: true
+    note: Dynamic input should be properly validated before usage.
```

Comment:
In what ways should the input be validated?

Reply:
Will update to include that in this case it was checking the input character set.